sql/planbuilder/scope.go

```diff
@@ -80,7 +80,7 @@ func (s *scope) resolveColumn(table, col string, checkParent bool) (scopeColumn,
 	var found scopeColumn
 	var foundCand bool
 	for _, c := range s.cols {
-		if strings.EqualFold(c.col, col) && (c.table == table || table == "") {
+		if strings.EqualFold(c.col, col) && (c.table == table || table == "") && (c.db == db || db == "") {
```

Comment:
i wonder if there's any weird behavior with col/db non-nil, but table nil....so like `select * from xy where mydb.``.x == 1`

Reply:
idk what this is doing, but it's doing the same thing as mysql
```sql
tmp> select db1.t1.i from db1.t1 where db1.``.i > 0;
+---+
| i |
+---+
| 1 |
+---+
1 row in set (0.00 sec)
```